content/docs/learn/typed-errors/from-either-to-raise.md

```diff
@@ -0,0 +1,181 @@
+---
+title: From Either to Raise
+sidebar_position: 5
+description: Learning about Raise from other FP ecosystems.
+---
+
+# From `Either` to `Raise`
+
+Typed errors in other functional ecosystems usually revolve around
+special types like `Either`, which provide the ability to describe
+computations that are successful or end in error. This style is
+fully supported by Arrow, but the DSL based around `Raise` usually
+results in nicer code. This small guide describes common patterns
+in the `Either` style and how they translate into `Raise`.
+
+In the following discussion, we assume the following functions:
+
+```kotlin
+fun f(n: Int): Either<Error, String>
+fun g(s: String): Either<Error, Thing>
+fun h(s: String): Either<Boo, Thing>
+fun Thing.summarize(): String
+```
+
+## Sequential composition
+
+The basic way to compose computations that may error out is by
+sequentially executing each of them, and finishing early if
+an error appears. This is done using `flatMap` — if the next
+computation may also fail — or `map` — if you need to apply
+a pure computation to the result, if available.
+
+Take this possible combination of the aforementioned functions:
+
+```kotlin
+fun tururu(n: Int): Either<Error, String> =
+  f(n).flatMap { s ->
+    g(s).map { t ->
+      t.summarize()
+    }
+  }
+```
+
+The translation into the `Raise` DSL describes the same sequence
+of steps, but using a more sequential style:
+
+```kotlin
+fun tururu(n: Int): Either<Error, String> = either {
+  val s = f(n).bind()
+  val t = g(s).bind()
+  t.summarize()
+}
+```
```

Comment:
This is the first mention of the "`Raise` DSL", and the code block after it... has the word `Either` twice, and `Raise` zero times.  This is very confusing.  Especially if I'm used to Haskell bind(), which is a bad alternate name for `flatMap()`, but I don't think that's the meaning here?